consent-ui/src/app/ais/entry-page/initiation/common/dedicated-access/dedicated-access.component.ts

```diff
@@ -51,9 +51,9 @@ export class DedicatedAccessComponent implements OnInit {
     consentObj.consent.access.availableAccounts = null;
     consentObj.consent.access.allPsd2 = null;
 
-    consentObj.consent.access.accounts = this.accounts.map(it => it.iban);
-    consentObj.consent.access.balances = this.accounts.map(it => it.iban);
-    consentObj.consent.access.transactions = this.accounts.map(it => it.iban);
+    consentObj.consent.access.accounts = this.accounts.map((it) => it.iban);
+    consentObj.consent.access.balances = this.accounts.map((it) => it.iban);
+    consentObj.consent.access.transactions = this.accounts.map((it) => it.iban);
```

Comment:
also it adds unneeded parentheses

Reply:
> also it adds unneeded parentheses

Parentheses are needed for the readability of the code in the IDE. For the automatic display of the object type within the parentheses

Reply:
then ok. I just noticed that default behaviour of prettier changed (arrow-parens default value changed from avoid to always in v2.0.0)